e2e/specs/suite/syncSuite.spec.ts

```diff
@@ -0,0 +1,200 @@
+import { logIn } from '../../pages/login.po';
+import { projectView } from '../../pages/project/view.po';
+import { suiteList } from '../../pages/suite/list.po';
+import { syncSuite } from '../../pages/modals/syncSuite.po';
+import { suiteView } from '../../pages/suite/view.po';
+import { ProjectHelper } from '../../helpers/project.helper';
+import { TestResult } from '../../../src/app/shared/models/test-result';
+import using from 'jasmine-data-provider';
+import cucumberImport from '../../data/import/cucumber.json';
+import syncImport from '../../data/import/oneTestCucumber.json';
+import users from '../../data/users.json';
+
+const editorExamples = {
+  localAdmin: users.localAdmin,
+  localManager: users.localManager,
+  localEngineer: users.localEngineer,
+  manager: users.manager
+};
+
+const notEditorExamples = {
+  viewer: users.viewer,
+  audit_admin: users.auditAdmin,
+  auditor: users.assignedAuditor
+};
+
+const isTestRunCorrect = (testResults: TestResult[], includedTests: string[], removedTests: string[]) => {
+  const tests = Array.from(testResults, testResult => testResult.test.name);
+  const included = includedTests.every((includedTest) => tests.includes(includedTest));
+  const removed = removedTests.every((removedTest) => !tests.includes(removedTest));
+  return included && removed;
+};
+
+const testSuite = 'Sync Test Suite';
+const defaultNumberOfTestRuns = '5';
+const numberOfTestRunsForSync = '2';
+const noSyncTest = 'Test Feature with all results: All passed';
+const syncTest = 'Test Feature with all results: step failed';
+const syncAndRemoveTest = 'Test Feature with all results: Step skipped';
+
+describe('Sync Test Suite', () => {
+  const projectHelper: ProjectHelper = new ProjectHelper();
+  const builds = projectHelper.generateBuilds(3);
+
+  beforeAll(async () => {
+    await projectHelper.init({
+      admin: users.admin,
+      localAdmin: users.localAdmin,
+      localManager: users.localManager,
+      localEngineer: users.localEngineer,
+      viewer: users.viewer
+    });
+  });
+
+  afterAll(async () => {
+    await projectHelper.dispose();
+  });
+
+  using(notEditorExamples, (user, description) => {
+    describe(`${description} role:`, () => {
+      beforeAll(async () => {
+        await logIn.logInAs(user.user_name, user.password);
+        await projectHelper.openProject();
+      });
+
+      it(`Sync suite is disabled for role ${description}`, async () => {
```

Comment:
rename to `Sync suite is not present for role ${description}`

Reply:
fixed